github/filecoin-project.yml

```diff
@@ -5089,8 +5086,10 @@ teams:
         - anorth
         - jennijuju
       member:
+        - arajasek
```

Comment:
```suggestion
```

Now that https://github.com/filecoin-project/FIPs/pull/991 has been merged, let's remove @arajasek again 😅